pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -795,8 +798,10 @@ func (cp *capacityPlugin) checkHierarchicalQueue(attr *queueAttr) error {
 
 		// Check if the parent queue's capability is less than the child queue's capability
 		if attr.capability.LessPartly(childAttr.capability, api.Zero) {
-			return fmt.Errorf("queue <%s> capability <%s> is less than its child queue <%s> capability <%s>",
-				attr.name, attr.capability, childAttr.name, childAttr.capability)
+			msg := fmt.Sprintf("Child queue %s capability (%s) exceeds parent capability (%s). "+
+				"Child's effective capability will be limited by parent.",
+				childAttr.name, childAttr.capability, attr.capability)
+			cp.recordQueueEvent(attr, ssn, v1.EventTypeWarning, "CapabilityExceeded", msg)
```

Comment:
I think we should not record event here, if this is matched, it will emit an event per batch duplicately

Reply:
I agree we should remove the event recording here.

Reply:
I think @hzxuzhonghu 's concern is that we will emit events every session, which is duplicate, but I still think record a event may help admin to debug questions, just as @guoqinwill said

Reply:
After adding the event recording, I found that misconfigured queues were generating too many events. I've added sorting for scalar resources to ensure consistent event messages, which helps with deduplication. However, even with identical events, Kubernetes still updates the Count and LastTimestamp fields, and requests are still sent to the API server on every scheduling cycle. I plan to use Prometheus metrics from the queue for monitoring and alerting instead.